### PR TITLE
Update prReview.txt - ADDED

### PR DESCRIPTION
**Pull request by GouriSurada3**
Update prReview.txt - ADDED

### CODE REVIEW

**Comment by GouriSurada3** [comment]
Adding normal comment

**Comment by GouriSurada3** [comment]
Adding second Comment on #40

**Comment by GouriSurada3** [comment]
Adding second Comment on https://github.com/GouriSurada3/Chessboard/pull/40

**Review by GouriSurada3 — COMMENTED** [review]
Need some more changes to be tested - edited one

**Review by GouriSurada3 — COMMENTED** [review]
Adding second review comment

File: prReview.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 PR Reviw\ew
+//Adding a commit
```
Comment (GouriSurada3) [inline prReview.txt:2]: add some more lines code

**Review by GouriSurada3 — COMMENTED** [review]
Inline comment + review comment adding -2

**Comment by GouriSurada3** [comment]
Adding First normal Comment on #40

**Review by GouriSurada3 — COMMENTED** [review]
Adding Finish your review normal comment -1

File: prReview.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 PR Reviw\ew
```
Comment (GouriSurada3) [inline prReview.txt:1]: add some more comments

**Review by GouriSurada3 — COMMENTED** [review]
Adding inline review comment

**Comment by GouriS101** [comment]
Adding comment from new user

**Review by bcgiri31 — COMMENTED** [review]
some review comment

**Comment by GouriSurada3** [comment]
New Comment on PR #40 - 3 sep 24

**Pull request closed without merging**